### PR TITLE
add documentation for jstor parser

### PR DESCRIPTION
**Pull request by joethei**
document new fetcher from [#6992](https://github.com/JabRef/jabref/pull/6992)

### CODE REVIEW

**Comment by mlep** [comment]
@Joethel We do not use separated help files for each database anymore. Please, copy-paste the information about the jstor database into https://github.com/JabRef/user-documentation/blob/master/en/collect/import-using-online-bibliographic-database.md

File: en/collect/import-using-online-bibliographic-database.md [hunk 1]
```diff
@@ -127,6 +127,17 @@ The following list shows some of the field indicators that can be used:
 * _a smith or a jones_: search for references with either author "smith" or author "jones"
 * _a smith and not t processor_: search for author "smith" and omit references with "processor" in the title
 
+### Jstor
+
+[Jstor](https://jstor.org) is an online databse with access to more than 12 million journal articles, books and sources in 75 disciplines. [About](https://about.jstor.org/)
+
+It is possible to limit the search by adding a field name to the search, as field:"text". The supported fields area:
```
Comment (koppor) [inline en/collect/import-using-online-bibliographic-database.md:134]:
```suggestion
It is possible to limit the search by adding a field name to the search, such as `field:"text"`. The supported fields are:
```

File: en/collect/import-using-online-bibliographic-database.md [hunk 2]
```diff
@@ -127,6 +127,17 @@ The following list shows some of the field indicators that can be used:
...
+
+* ti: The title of the article
+* au: a author of the article
+* pt: publication title
```
Comment (koppor) [inline en/collect/import-using-online-bibliographic-database.md:138]:
```suggestion
* `ti`: The title of the article
* `au`: a author of the article
* `pt`: publication title
```

**Review by koppor — CHANGES_REQUESTED** [review]
Some nitpicks regarding markdown.

**Review by mlep — CHANGES_REQUESTED** [review]

**Pull request merged**